widgets/class-monk-language-switcher.php

```diff
@@ -88,7 +88,7 @@ public function widget( $args, $instance ) {
 							$current_url = is_ssl() ? 'https://' . $current_url : 'http://' . $current_url;
 						} else {
 							if ( dirname( $_SERVER['PHP_SELF'] ) === '\\' || dirname( $_SERVER['PHP_SELF'] ) === '/' ) {
-								$current_url = trailingslashit( $current_url ) . $lang_code;
+								$current_url = str_replace( home_url(), home_url() . $lang_code . '/', $current_url );
```

Comment:
This happens when we have the URL without a language, but we need to put a URL with language in the other links on language switcher, so we need to add the language.